### ISSUE TITLE
Div is missing accessible name

### ISSUE THREAD

**Issue by vh8154** [pulibrary/DSS#472]
### Steps to recreate
1. Go to https://dss.princeton.edu/regions
2. Log in
3. Visit [this page](https://dss.princeton.edu/regions)
4. click "edit" for one of the rows
5.  click "Update Region".
6. Run axe DevTools on this page.

### URL
https://dss.princeton.edu/regions

### Issues
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

### Severity
Moderate

### WCAG Success Criterion
[WCAG 1.3.6: Identify Purpose](https://www.w3.org/WAI/WCAG22/Understanding/identify-purpose)

### WCAG Conformance level (A, AA, AAA)
AAA

### User impact
The purpose of this element on a page is hard to be programmatically determined.

### Recommended fix
The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable.

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
